README.rst

```diff
@@ -258,6 +258,34 @@ For accounts in regions outside of US-West, use :code:`region` parameter to spec
     $dbh = new PDO("snowflake:account=testaccount.us-east-1", "user", "password");
     $dbh = new PDO("snowflake:account=testaccount;region=us-east-1", "user", "password");
 
+Using Key Pair Authentication
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The PHP PDO driver supports key pair authentication and key rotation.
+
+You must first complete the initial configuration for key pair authentication as shown in 
+`Key Pair Authentication & Key Pair Rotation <https://docs.snowflake.com/en/user-guide/key-pair-auth.html#key-pair-authentication-key-pair-rotation>`_.
+
+To connect to the Snowflake database using key par authentication, create a new :code:`PDO` object, as explained in the
+`PHP PDO documentation <https://www.php.net/manual/en/pdo.connections.php>`_.
+Specify the data source name (:code:`dsn`) parameter as shown below:
+
+.. code-block:: php
+
+    $dbh = new PDO("account=<account name>;authenticator=SNOWFLAKE_JWT;priv_key_file=<path>/rsa_key.p8;priv_key_file_pwd=<private_key_passphrase>", 
+                    <username>, "");
```

Comment:
`<username>` should be `"<username>"`, as it is on line 244.

Reply:
Done